todo_app/flask_config.py

```diff
@@ -4,12 +4,8 @@
 class Config:
     def __init__(self):
         """Base configuration variables."""
-        #self.SECRET_KEY = os.environ.get('SECRET_KEY')
-
-        self.T_KEY = os.environ.get('T_KEY')
-        
-        self.T_TOKEN = os.environ.get('T_TOKEN')
+        self.M_KEY = os.environ.get('M_KEY')
         
-        self.TRELLO_BOARD_ID=os.environ.get('TRELLO_BOARD_ID')
+        self.M_TOKEN = os.environ.get('M_TOKEN')
         
-        self.TRELLO_BOARD_NAME = os.environ.get('TRELLO_BOARD_NAME')
\ No newline at end of file
+        self.MONGO_BOARD_ID=os.environ.get('MONGO_BOARD_ID')
```

Comment:
Do you ever use these from the config, or just access them fresh as environment variables? Not an issue, but might be easier to just remove them from here